### ISSUE TITLE
[BUG] Billing and Transfer Money to another account.

### ISSUE THREAD

**Issue by alexzh16** [iNextrix/ASTPP#145]
Hi, in Astpp its very big bug in billing system. i think ver.3 have same bug.
1)  create two account, for example user1 and user2
2) recharge user1 with 5usd.
3) make calling with user1 to destination 1 (client in another site answer and talking)
4) in this moment user1 transfer all money 5usd to account user2, in this moment user1 have 0 in account and user2 have 5usd. user1 talking about 24minuts free.

can you make change in billing system? if user1 calling, or start calling, in moment transfer money to another account, check if, client have active process and reject transfer money.

sory for my english.

thx. Alex.

**Comment by dedayoa** [comment]
Ah, it took me a while, but I understand what you describe. This may be because the cost for the call is not deducted until the end of the call.
I'm interested to see how this is solved.
But how exactly does one transfer money from one customer (account) to another?

**Comment by alexzh16** [comment]
yes, the cost for the call is deducted only from start call and say where max minutes you can calling. After end call, is deducted sum. But in this moment, in this account balance is 0. This fraud calling I'm met in my sip server. Unfortunately people from India calling many time with this BUG. In ASTPP if you have open two account - you have number of this accounts and you can transfer money from one to another accounts. You must close this possibility in astpp site.

**Comment by alexzh16** [comment]
yes, its good idea with lock, unlock users.

**Comment by smrdoshi** [comment]
We will probably set lock and unlock account concept to prevent this issue. If customer is on call then account will marked as locked and will not be able to do transfer money operation. We may use lock / unlock concept for some other features. 

If anyone have other idea then share with us.

**Comment by Hemdip** [comment]
@alexzh16 It will be fixed in next release
